starwars/src/App.js

```diff
@@ -1,19 +1,56 @@
-import React from 'react';
-import './App.css';
+import React, { useEffect, useState } from "react";
+import "./App.css";
+import axios from "axios";
+import Character from "./components/Character";
 
 const App = () => {
+  const [data, setData] = useState([]);
+  // const [searchResult, setSearchResult] = useState("");
+
+  useEffect(() => {
+    axios
+      .get("https://swapi.dev/api/people")
+      .then((res) => {
+        // console.log(res.data.results);
+        const dataArr = res.data.results;
+        setData(dataArr);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }, []);
+
+  console.log(data);
+
   // Try to think through what state you'll need for this app before starting. Then build out
   // the state properties here.
 
-  // Fetch characters from the API in an effect hook. Remember, anytime you have a 
+  // Fetch characters from the API in an effect hook. Remember, anytime you have a
   // side effect in a component, you want to think about which state and/or props it should
   // sync up with, if any.
 
   return (
     <div className="App">
       <h1 className="Header">Characters</h1>
+
```

Comment:
You can pass down one prop here... <Character characters={characters} /> 
...and then change your Character file to match the props being passed down